src/component/Footer.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import styled from "styled-components";
+import Moto from "../icons/moto";
```

Comment:
Make sure that components and file names are both in PascalCase. `moto`, `chat` and `profile` filenames should have an uppercase first character. there might be more components/files with the same issue.